src/lib/support/value_parser/ValueParserIntegral.cpp

```diff
@@ -0,0 +1,101 @@
+/*
```

Comment:
I don't understand why this is ValueParserIntegral.   Is this supposed to be ValueParserScalar, maybe?  It certainly parses not just integral values....